docs/editing.md

```diff
@@ -174,6 +184,20 @@ This doesn’t mean that you’re the editor, just that you’ve been suggested
 
 ![Editorial flow](images/JOSS-flowchart.png)
 
+## Expectations on JOSS editors
+
+### Responding to editorial assignments
+
+As documented above, papers will be assigned to you by one of the EiCs. We ask that editors respond in a timely fashion (< 72 hours) to invites to edit a new submission.
```

Comment:
As documented above, papers will be assigned to you by one of the AEiCs. We ask that editors respond in a timely fashion (< 72 hours) to invites to edit a new submission.